### PR TITLE
Add securityContext configuraton to helm templates of sra and api-gw

### PR DESCRIPTION
**Pull request by tapuhi**

### CODE REVIEW

File: charts/akeyless-api-gateway/templates/deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,11 @@ spec:
       labels:
         {{- include "akeyless-api-gw.selectorLabels" . | nindent 8 }}
     spec:
+      {{- if .Values.deployment.securityContext.enabled }}
+      securityContext:
+         runAsUser: {{ .Values.deployment.securityContext.runAsUser }}
+         fsGroup: {{ .Values.deployment.securityContext.fsGroup }}
+      {{- end }}
```
Comment (OriBenHur-akeyless) [inline charts/akeyless-api-gateway/templates/deployment.yaml:27]: What are those uid and gid?

File: charts/akeyless-api-gateway/values.yaml [hunk 2]
```diff
@@ -33,7 +33,10 @@ deployment:
   nodeSelector:
     #     iam.gke.io/gke-metadata-server-enabled: "true"
 
-
+  securityContext:
+    enabled: false
+    fsGroup: 1001
+    runAsUser: 1001
```
Comment (OriBenHur-akeyless) [inline charts/akeyless-api-gateway/values.yaml:39]: Same

File: charts/akeyless-secure-remote-access/values.yaml [hunk 3]
```diff
@@ -183,7 +183,10 @@ sshConfig:
   #      eks.amazonaws.com/role-arn: arn:aws:iam::40XXXXXXXX75:role/eksctl-Role1-XXXXXXXX
   nodeSelector:
   #     iam.gke.io/gke-metadata-server-enabled: "true"
-
+  securityContext:
+    enabled: false
+    fsGroup: 1001
+    runAsUser: 1001
```
Comment (OriBenHur-akeyless) [inline charts/akeyless-secure-remote-access/values.yaml:189]: Same

File: charts/akeyless-api-gateway/values.yaml (hunk 2 above)
Comment (tapuhi) [reply charts/akeyless-api-gateway/values.yaml:39]: Just an example . I will put there 0 for root

File: charts/akeyless-secure-remote-access/values.yaml (hunk 3 above)
Comment (tapuhi) [reply charts/akeyless-secure-remote-access/values.yaml:189]: Just an example . I will put there 0 for root

File: charts/akeyless-api-gateway/templates/deployment.yaml (hunk 1 above)
Comment (tapuhi) [reply charts/akeyless-api-gateway/templates/deployment.yaml:27]: Here the uid is comming from the values file

**Review by OriBenHur — APPROVED** [review]

**Review by OriBenHur-akeyless — APPROVED** [review]

**Review by liorhasson — APPROVED** [review]

**Review by OriBenHur-akeyless — APPROVED** [review]

**Pull request merged**